app/controllers/api/v1/courses_controller.rb

```diff
@@ -1,8 +1,25 @@
 module Api
   module V1
     class CoursesController < BaseController
+      require 'lms_api'
```

Comment:
What do we use this for?  I thought we were going to try to use the canvas facade

Reply:
Also, how are we making the call to canvas?